### PR TITLE
added optional param table_name to sorting_order method

### PR DESCRIPTION
**Pull request by matiascondem**
When working with STI or long name models, the table's name differs from what a model.tableize would be. This changes allows you to pass the custom table's name to the sorting_order method.

### CODE REVIEW

File: lib/sort_n_params/concerns/scopes.rb [hunk 1]
```diff
@@ -4,15 +4,17 @@ module SortNParams
   module Scopes
     extend ActiveSupport::Concern
     class_methods do
-      def sorting_order(ordering_array)
+      def sorting_order(ordering_array, table_name)
```
Comment (JuannFerrari) [inline lib/sort_n_params/concerns/scopes.rb:7]: Could you change this to `table_name = nil`? I think that it has to be an optional param

**Comment by matiascondem** [comment]
Glad you liked it! In my last commit I added test cases. Check it out!

File: spec/concerns/scopes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  let(:test_class) do
+    CustomTable = Struct.new(:custom_field) do
+      include SortNParams::Scopes
+    end
+  end
+
+  context 'when you pass only an ordering array' do
+    subject { test_class.parse_array(["test_field","asc"]) }
+
+    it "builds the array correctly with the table_name param passed" do
```
Comment (JuannFerrari) [inline spec/concerns/scopes_spec.rb:13]: We may have to change this to:

`it "builds the array correctly without the table_name param passed" do`

**Comment by JuannFerrari** [comment]
Great job! 🚀

**Pull request merged**